services/api-dotnet/src/app/Api.NoSql/appsettings.Development.json

```diff
@@ -24,5 +24,12 @@
     "ConnectionString": "redis:6379", // Docker
     "AbsoluteExpirationInSeconds": 600,
     "SlidingExpirationInSeconds": 60
+  },
+  "CloudStorage": {
+    "Endpoint": "",
+    "BucketName": "",
+    "FolderName": "",
+    "AccessKey": "",
+    "Secret": ""
```

Comment:
Let's move the whole section up into appsettings.json for consistency. And also replace empty strings with nulls (there's no particular difference - just for consistency).